README.md

```diff
@@ -42,50 +49,100 @@ PRETTYLOG 0
 FILESTAMP 1
 LOGFILELEVEL 3
 ```
-Flag         | Type  | Default | Internal Name  | Description
--------------|-------|---------|----------------|------------
-LOGLEVEL     | level | lvl     | slg.level      | Max level to print to stdout
-LOGTOFILE    | bool  | 0       | slg.to_file    | If 0 will not write to file
-PRETTYLOG    | bool  | 0       | slg.pretty     | If 1 will output with color
-FILESTAMP    | bool  | 1       | slg.filestamp  | If 1 will add date to log name
-LOGFILELEVEL | level | flvl    | slg.file_level | Level required to write to file
+Flag         | Default | Internal Name  | Description
+-------------|---------|----------------|------------
+LOGLEVEL     | lvl*    | slg.level      | Max level to print to `stdout`.
+LOGTOFILE    | 0       | slg.to_file    | If 0 will not write to file.
+PRETTYLOG    | 0       | slg.pretty     | If 1 will output with color.
+FILESTAMP    | 1       | slg.filestamp  | If 1 will add date to log name.
+LOGFILELEVEL | lvl**   | flvl           | slg.file_level | Level required to write to file.
+
+**Note:** `LOGFILELEVEL` and `LOGLEVEL` are completely independent of each other.
 
-LOGFILELEVEL and LOGLEVEL are completly independent of each other.
+**`LOGLEVEL` is passed as third argument in `slog_init()`.*   
+**`LOGFILELEVEL` is passed as fourth argument in `slog_init()`.*
 
 ### Logging flags
 Slog has its logging flags to print something with status code.
 
-- SLOG_LIVE
```

Comment:
Let's additionally keep up with legacy flags. Please check my comment in slog.h line 48
Maybe add something along:

"As of version 1.5 the flags have changed. In order to keep backwards-compatibility, the legacy flags can be used as usual:"

- `SLOG_LIVE`
- `SLOG_INFO`
- `SLOG_WARN`
- `SLOG_DEBUG`
- `SLOG_ERROR`
- `SLOG_FATAL`
- `SLOG_NONE`